airflow/utils/db_cleanup.py

```
@@ -159,6 +170,14 @@ def _do_delete(*, query, orm_model, skip_archive, session):
     logger.debug("delete statement:\n%s", delete.compile())
     session.execute(delete)
     session.commit()
+    if export_to_csv:
+        if not output_path.startswith(AIRFLOW_HOME):
+            output_path = os.path.join(AIRFLOW_HOME, output_path)
```

We shouldn't restrict this to AIRFLOW_HOME.

---

yeah, probably we should just accept whatever path the user has requested.  i guess depending on whether we want to create the parent dirs for the user if they don't exist, we could either do so, or fail fast if they don't